docs/reference/csharp/v0/api/api-delete.md

```diff
@@ -0,0 +1,110 @@
+---
+title: api.delete()
+description: Register an API route and set a specific HTTP DELETE handler on that route.
+---
+
+Register an API route and set a specific HTTP DELETE handler on that route.
+
+> This method is a convenient short version of [api().route().delete()](./api-route-delete)
+
+```c#
+using Nitric.Sdk;
+
+var api = Nitric.Api("main");
+
+api.Delete("/hello/:name", context => {
+  var name = context.Req.PathParams.get("name");
+
+  context.Res.Text($"Deleting {name}!");
+
+  return context;
+});
+```
```

Comment:
should we include `Nitric.Run()` in these examples?

Just a question, as it might be easy to miss.